### ISSUE TITLE
Bandwidth limit? 

### ISSUE THREAD

**Issue by ser** [vog/bscp#7]
Any suggestion how one can limit bandwidth?

**Comment by vog** [comment]
This would be a nice feature, so any volunteer work on this topic is welcome.

Without changing `bscp`, this is perhaps also possible using advanced packet filter settings. This is cetainly possible on Linux as well as BSD, although I'm not too much into it to provide a working configuration here.

**Comment by ser** [comment]
I think it would be as easy as adding "pv -R" into the pipe before invoking ssh, but to be honest I have no idea how to technically incorporate it into python's subprocess module.

**Comment by vog** [comment]
I don't think it is that easy, as there is the whole protocol of exchanging and comparing block checksums that needs to be reversed.

Also, it is not obvious to me which parts of the code can be used for both directions, and which parts are specific for one direction.

Anyway, feel free to give it a try. If it works, open a pull request.

**Comment by vog** [comment]
To anyone involved in this issue, please see #16 "Future of bscp".

**Comment by vog** [comment]
Pull requests on this issue are still welcome.
